Rubberduck.UnitTesting/CodeGeneration/TestCodeGeneratorStatics.cs

```diff
@@ -116,12 +116,12 @@ End If
 #End If";
 
         private static string TestModuleTemplate =>
-$@"{{0}}Option Explicit
-Option Private Module
-
```

Comment:
The empty line is expected in the test that broke; either the empty line needs to stay, or the test expectations needs to be updated to reflect the template changes.